packages/juggler/test/unit/legacy/legacy.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: @loopback/juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+
+import {jugglerModule, bindModel, DataSource, juggler, DefaultCrudRepository}
+  from '../../../src/legacy';
+
+describe('legacy loopback-datasource-juggler', () => {
+  let ds: juggler.DataSource;
+
+  before(function() {
+    ds = new DataSource({
+      name: 'db',
+      connector: 'memory',
+    });
+    expect(ds.settings.name).to.eql('db');
+    expect(ds.settings.connector).to.eql('memory');
+  });
+
+  it('creates models', () => {
+    /* tslint:disable:variable-name */
```

Comment:
Please use `tslint:disable-next-line:variable-name`. IIUC, your version disables the check for the rest of the file. This applies to most (if not all) other usages of `tslint:disable` in this patch.